test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.common;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class UtilsTest {
+	
+	@Before
+	public void setup() {
+		
+	}
+	
+	
+	/**
+	 * Test isAnyNull()
+	 */
+	
+	@Test
+	public void onenull_returnsTrue() {
```

Comment:
Hi Hi, try to camel case your naming, do refer to the java coding standards for more details.